render/tests/test-component.js

```diff
@@ -4,7 +4,7 @@ var o = require("../../ospec/ospec")
 var domMock = require("../../test-utils/domMock")
 var vdom = require("../../render/render")
 
-o.spec("component", function() {
+o.spec("component object", function() {
```

Comment:
This is the only real change. The rest is whitespace noise (an EditorConfig would be nice).